x/concentrated-liquidity/tick_test.go

```diff
@@ -132,26 +132,28 @@ func (suite *KeeperTestSuite) TestTickToSqrtPrice() {
 		name              string
 		tickIndex         sdk.Int
 		sqrtPriceExpected string
-		expectErr         bool
 	}{
 		{
-			"happy path",
+			"happy path 1",
 			sdk.NewInt(85176),
-			"70.710004849206351867",
-			false,
+			"70.710004849206351861", // 70.710004849206120647
+			// https://www.wolframalpha.com/input?i2d=true&i=Power%5B1.0001%2CDivide%5B85176%2C2%5D%5D
+		},
+		{
+			"happy path 2",
+			sdk.NewInt(86129),
+			"74.157016832801088770", // 74.160724590950847046
```

Comment:
I think the problem unfortunately exists for odd powers.

From python:

```
>>> 1.0001**(43064.5)
74.16072459091568
>>> 1.0001**(43064)
74.15701683276568
```

It seems that here we round down when dividing. That's where we lose a lot of precision

Reply:
Good catch, we for sure have to do something about the odd ones at the very least, would like to improve precision all around if at all possible

Reply:
Maybe for odd we use `ApproxRoot` for now?

I can look into implementing a more suitable root or power functions later

Reply:
Ended up going back to using ApproxRoot for both for now